articles/quickstart/webapp/aspnet-core-blazor-server/interactive.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Add Login to your ASP.NET Core Blazor Server application
+description: This tutorial demonstrates how to add user login to an ASP.NET Core Blazor Server application.
+budicon: 448
+topics:
+  - quickstarts
+  - webapp
+  - aspnet-core
+  - blazor-server
+  - login
+github:
+  path: Quickstart/Sample
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/login-model
+  - files/logout-model
+  - files/profile
+  - files/program
+---
+
+# Add Login to your Blazor Server application
+
+Auth0 allows you to quickly add authentication and gain access to user profile information in your application. This guide demonstrates how to integrate Auth0 with any new or existing Blazor Server application using the **Auth0.AspNetCore.Authentication** SDK. 
+
+<%= include('../../_includes/_configure_auth0_interactive', { 
+  callback: 'http://localhost:3000/callback',
+  returnTo: 'http://localhost:3000'
+}) %>
+
+## Install and Configure the SDK {{{ data-action=code data-code="Program.cs" }}}
+
+### Install from Nuget
+
+To integrate Auth0 with Blazor Server you can use our SDK by installing the `Auth0.AspNetCore.Authentication` [Nuget package](https://www.nuget.org/packages/Auth0.AspNetCore.Authentication/) to your application.
+
+```bash
+Install-Package Auth0.AspNetCore.Authentication
+```
+
+### Configure the middleware
+
+To enable authentication in your Blazor Server application, use the middleware provided by the SDK. Go to the `Program.cs` file and call `builder.Services.AddAuth0WebAppAuthentication()` to register the SDK's middleware.
+
+Ensure to configure the `Domain` and `ClientId`, these are required fields to ensure the SDK knows which Auth0 tenant and application it should use.
+
+Make sure you have enabled authentication and authorization in your `Program.cs` file.
+
+## Login {{{ data-action=code data-code="Login.cshtml.cs" }}}
+
+To allow users to login to your Blazor Server application, add a `LoginModel` to your `Pages` directory.
+
+Inside the `LoginModel`'s `OnGet` method, call `HttpContext.ChallengeAsync()` and pass `Auth0Constants.AuthenticationScheme` as the authentication scheme. This will invoke the OIDC authentication handler that our SDK registers internally. Be sure to also specify the corresponding `authenticationProperties`, which you can construct using the `LoginAuthenticationPropertiesBuilder`.
+
+After succesfully calling `HttpContext.ChallengeAsync()`, the user will be redirected to Auth0 and signed in to both the OIDC middleware and the cookie middleware upon being redirected back to your application. This will allow the users to be authenticated on subsequent requests.
```

Comment:
```suggestion
After successfully calling `HttpContext.ChallengeAsync()`, the user will be redirected to Auth0 and signed in to both the OIDC middleware and the cookie middleware upon being redirected back to your application. This will allow the users to be authenticated on subsequent requests.
```